spec.html

```diff
@@ -26406,7 +26277,7 @@ <h1>
             _referencingScriptOrModule_: unknown,
             _specifier_: unknown,
             _promiseCapability_: a PromiseCapability Record,
-            _completion_: unknown,
+            _completion_: a normal completion whose [[Value]] is *undefined*,
```

Comment:
Nope; the assertion below was conditional. You can make this `a Completion Record`, but you do need to retain the assertion below.